packages/backend/src/studio.ts

```diff
@@ -126,6 +127,7 @@ export class Studio {
         const href = link.match(/href="(.*?)"/);
         if (href) {
           const webviewHref = this.#panel?.webview.asWebviewUri(Uri.joinPath(extensionUri, 'media', href[1]));
+          if (!webviewHref) throw new Error('undefined webviewHref');
```

Comment:
I think you might need to add more context in the error message for this one and the one line 117 because if I see the error, I don't know for which link for example we tried to find the webviewHref, etc